src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46839/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46839/project/issues?id=sample&pullRequest=5750&issues=AWdgRLv5b_4MKAjqKtLb&open=AWdgRLv5b_4MKAjqKtLb)

[ISSUE_KEY]: AWdgRLv5b_4MKAjqKtLb